doc/classes/RegExMatch.xml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="RegExMatch" inherits="Reference" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		[RegExMatch] is a result container for [RegEx].
 	</brief_description>
 	<description>
+		A [RegExMatch] is returned by [method RegEx.search] if the subject match with the compiled pattern of [RegEx].
```

Comment:
match -> matches?

Reply:
I don't know because it's the first match wich is returned.

Reply:
Please avoid passive form too: 
[method RegEx.search] returns a [RegExMatch] if the subject matches with compiled pattern of [RegEx]

Reply:
@groud passive is fine here as the class is manipulated by another method. It could be more precise though.

Things to review:
- don't add links to the current class (generates hyperlinks to the current page), use [code] instead, e.g. `[code]RegExMatch[/code]`
- "subject" is super confusing to me. I see it's in the API, but I think it's not clear at all. I'd recommend clearer words like search pattern, [search] match(es) and [source] string. Other devs, what do you think?

Suggestion:
`Contains the results of a regex search. [method RegEx.search] returns an instance of [code]RegExMatch[/code] if it finds the search pattern in a string.`

You may replace search pattern by the corresponding member variable from one of the regex classes? It doesn't work yet but [member ...] should eventually work like [method ...]

Reply:
The `pattern` and `subject` were based off Python's naming conventions in the re module, which was what I was testing the original regex library against.